projects/ccf-rui/src/app/modules/info/info-dialog/info-dialog.component.html

```diff
@@ -5,73 +5,11 @@
-          3D reference organs are freely available on the <a class="link-blue" rel="noopener noreferrer" href="https://hubmapconsortium.github.io/ccf/pages/ccf-3d-reference-library.html" target="_blank">CCF Portal</a>,
-          the ontology can be found on
-          <a class="link-blue" rel="noopener noreferrer" href="https://bioportal.bioontology.org/ontologies/CCF/" target="_blank">BioPortal</a>,
-          and code is available on
-          <a class="link-blue" rel="noopener noreferrer" href="https://github.com/hubmapconsortium/ccf-ui/" target="_blank">GitHub</a>.
-          A demo video is on YouTube
-          <a class="link-blue" rel="noopener noreferrer" href="https://youtu.be/z1aOjHFetME" target="_blank">here</a>.
-        </p>
-
-        <p class="description">
-          The registration data is used in current versions of the Common Coordinate Framework (CCF, see
-          <a class="link-blue" rel="noopener noreferrer" href="https://hubmapconsortium.github.io/ccf/" target="_blank">CCF Portal</a>
-          for details) and the
-          <a class="link-blue" rel="noopener noreferrer" href="https://portal.hubmapconsortium.org/ccf-eui" target="_blank">CCF Exploration User Interface (EUI)</a>
-          developed within HuBMAP.
-        </p>
-
-        <br />
-
-        <h2>Acknowledgements</h2>
-        <p class="description">
-          The CCF and associated user interfaces are developed by the MC-IU team within the HuBMAP HIVE. The MC-IU team includes Katy Börner, Lisel Record, Bruce Herr II, Leonard Cross, Ellen Quardokus, and Andreas Bueckle, Indiana University, Bloomington, IN and Griffin Weber, Harvard Medical School, Boston, MA. Former members of the team are Paul Macklin, Randy Heiland, and Jim Sluka at Indiana University, Bloomington, IN and Samuel Friedman, Opto-Knowledge Systems, Inc. We acknowledge expert input from Jeffrey Spraggins, Sanjay Jain, and Clive Wasserfall and their teams as well as the overall HuBMAP consortium.
-        </p>
-
-        <p class="description">
-          The 3D reference organs were designed by Kristen Browne, U.S. National Library of Medicine using data from the
-          <a class="link-blue" rel="noopener noreferrer" href="https://www.nlm.nih.gov/research/visible/visible_human.html" target="_blank">Visible Human Project</a>.
-        </p>
-
-        <p class="description">
-          The work is funded by NIH Award <a class="link-blue" rel="noopener noreferrer" href="https://projectreporter.nih.gov/project_info_description.cfm?aid=9687220" target="_blank">OT2OD026671</a>.
-        </p>
+      
+      <mat-dialog-content class="mat-typography dialog-content">
+        <div *ngFor="let content of documentationComtents; let i = index;" class="panel">
+          <ccf-info-panel [content]="content" [index]="i"></ccf-info-panel>
```

Comment:
You sure you want to break this out as a separate component vs just using the little bit of code used in ccf-info-panel directly here?

Reply:
Yeah you're right. I've refactored it